cmd/acra-keys/keys/export_test.go

```diff
@@ -0,0 +1,274 @@
+package keys
+
+import (
+	"encoding/base64"
+	"flag"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/cossacklabs/acra/keystore"
+	"github.com/cossacklabs/acra/keystore/filesystem"
+	"github.com/cossacklabs/acra/keystore/keyloader"
+	"github.com/cossacklabs/acra/keystore/keyloader/env_loader"
+)
+
+func TestExport_Import_CMD_FS_V1(t *testing.T) {
```

Comment:
we test here correct flow, but lets test incorrect flow too. Try to create random file named as private key in valid keystore folder, or specify folder with simple files but not valid keys.